src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -155,8 +162,11 @@ const SendFunds = ({
       ),
     )(amount)
 
+    const addressValidation = composeValidators(minMaxLength(1, 42))(recipientAddress)
```

Comment:
The QR code feature apparently skips this validation.

Btw, maybe we could use [`Web3.utils.isAddress`](https://web3js.readthedocs.io/en/v1.3.4/web3-utils.html#isaddress)?